protocol/message.go

```diff
@@ -30,6 +31,13 @@ type Message struct {
 	// routes that match the filters
 	Filters map[string]string
 
+	// Expires field specifies until when the message is valid to be processed
+	// If this field is set and the message is expired the connectors should
+	// consider the message as processed and log the action
+	//
+	// RFC3339 format
+	Expires *time.Time
```

Comment:
Sorry for me asking, but why are you requiring the timezone information? Wouldn't it suffice to know the exact time the message expires? Which is given by a unix timestamp (since it is in UTC).